app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -302,6 +302,8 @@ def pod_definition
   end
 
   def inspector_image
-    'docker.io/openshift/image-inspector:v1.0.z'
+    registry = @ems_configs.fetch_path(:ems_kubernetes, :image_inspector_registry)
+    return "#{registry}/openshift/image-inspector:v1.0.z" unless registry.empty?
```

Comment:
@dtrieu80 I'd make the `registry` part (or whatever name would be nicer) be `docker.io/openshift` ...since it seems more configurable.

    return "#{registry}/image-inspector:v1.0.z" unless registry.empty?

Then maybe you would like to make a constant out of `image-inspector:v1.0.z` since you're using it twice.